### PR TITLE
Remove sophtron and mx code/tests from apps/server

### PR DESCRIPTION
**Pull request by tphalp**

### CODE REVIEW

File: apps/server/src/test/testData/institution.ts [hunk 1]
```diff
@@ -4,32 +4,6 @@ import {
...
-};
-
-export const sophtronUserInstitutionAccountsData = [
-  {
-    AccountName: "AccountName",
-    AccountNumber: "AccountNumber",
-    AccountID: "AccountID",
-  },
-];
-
```
Comment (tphalp) [inline apps/server/src/test/testData/institution.ts:32]: These were all unused

File: apps/server/src/utils/index.ts [hunk 2]
```diff
@@ -1,56 +1,6 @@
-import { createCipheriv, createDecipheriv } from "crypto";
```
Comment (tphalp) [inline apps/server/src/utils/index.ts:1]: More unused methods/vars in this file

File: apps/server/src/adapterSetup.ts [hunk 3]
```diff
@@ -1,25 +1,28 @@
...
-    widgetAdapter: new SophtronAdapter(),
-  },
-};
+const sophtronAdapterMapObject: Record<string, AdapterMap> =
+  getSophtronAdapterMapObject({
+    logClient: logger,
+    aggregatorCredentials: {
+      clientId: config.SophtronApiUserId,
+      secret: config.SophtronApiUserSecret,
+      endpoint: config.SophtronApiServiceEndpoint,
```
Comment (wesrisenmay-mx) [inline apps/server/src/adapterSetup.ts:17]: should endpoint and vcEndpoint just be hardcoded into the adapter?

Comment (tphalp) [reply apps/server/src/adapterSetup.ts:17]: Good catch.

Comment (tphalp) [reply apps/server/src/adapterSetup.ts:17]: We should do the same for the MX adapter.

**Comment by wesrisenmay-mx** [comment]
It's nice to see all this red

**Review by wesrisenmay-mx — APPROVED** [review]

**Pull request merged**